documentation/milestones/milestone_short_descriptions.md

```diff
@@ -290,6 +299,19 @@ Make a plan for the technological and legal/process aspects of A/B testing exper
 
 This plan will need to account for any concerns about fairness that could be brought up by exposing some users to updated content as compared to other users.
 
+## API security planning
+
+Diagram short name: `API-Security`
+
+Dependencies: `DB-API-Plan`, `GET-Opportunities`
+
+Develop a plan for securing the public API. While the broader AuthN and AuthZ framework will be designed and implemented in a different set of milestones, this plan should account for applying basic security measures to the initial set of endpoints we will be deploying, namely `GET /opportunities`
```

Comment:
nit: missing a period at the end of the sentence

Reply:
Good catch! Just pushed up a change for this!